charts/datadog/templates/container-process-agent.yaml

```diff
@@ -26,10 +26,8 @@
     {{- end }}
     - name: DD_LOG_LEVEL
       value: {{ .Values.agents.containers.processAgent.logLevel | default .Values.datadog.logLevel | quote }}
-    {{- if or .Values.datadog.systemProbe.enabled .Values.datadog.networkMonitoring.enabled }}
     - name: DD_SYSTEM_PROBE_ENABLED
-      value: {{ true | quote }}
-    {{- end }}
+      value: {{ (or .Values.datadog.systemProbe.enabled .Values.datadog.networkMonitoring.enabled) | quote }}
```

Comment:
should this "or" sequence look like https://github.com/DataDog/helm-charts/pull/101/files#diff-483dad0ace7ed469efb9d93ba2f3d3c2045bbc1cd317f49e7e0a122bdf9d6116R194 , or does it not matter?

Reply:
We should only set the DD_SYSTEM_PROBE_ENABLED=true if the process-agent is enabled. The reason is that prior to 7.24, DD_SYSTEM_PROBE_ENABLED controlled network monitoring and only network monitoring. Past 7.24 there are a new set of feature specific flags that we can rely on.  This seemed like the best way to have the chart continue to work with agent versions lower than that.

Reply:
ah, interesting. okay thanks for explaining